packages/arb-avm-cpp/data_storage/include/data_storage/datastorage.hpp

```diff
@@ -85,9 +85,9 @@ class Transaction {
         : datastorage(std::move(datastorage_)),
           transaction(std::move(transaction_)) {}
 
-    rocksdb::Status commit() { return transaction->Commit(); }
+    rocksdb::Status commit() const { return transaction->Commit(); }
 
-    rocksdb::Status rollback() { return transaction->Rollback(); }
+    rocksdb::Status rollback() const { return transaction->Rollback(); }
```

Comment:
I kinda like these being non-const for semantic reasons

Reply:
sounds good